sup3r/pipeline/forward_pass.py

```diff
@@ -803,6 +972,17 @@ def __init__(self, strategy, node_index=0):
         self.compute_workers = kwargs['compute_workers']
         self.load_workers = kwargs['load_workers']
         self.output_workers = kwargs['output_workers']
+        self.exo_kwargs = kwargs['exo_kwargs']
+        self.exo_slices = kwargs['exo_slices']
+
+        if self.exo_kwargs is not None:
+            self.exo_features = self.exo_kwargs.get('features', [])
+            self.features = [f for f in self.features
+                             if f not in self.exo_features]
+            self.exo_handler = ExogenousDataHandler(**self.exo_kwargs)
```

Comment:
You probably dont need to assign the handler to an attribute since it's not used elsewhere, but mostly you just want to watch out for conditional attributes - this attribute does not exist at all if exo_kwargs is None